Chapter1/7-Transformations/Window.cs

```diff
@@ -89,6 +77,15 @@ protected override void OnLoad()
             GL.EnableVertexAttribArray(texCoordLocation);
             GL.VertexAttribPointer(texCoordLocation, 2, VertexAttribPointerType.Float, false, 5 * sizeof(float), 3 * sizeof(float));
 
+            _texture = new Texture("Resources/container.png");
```

Comment:
hmm this might be new code for Texture..... id be a little more happy if we could use a factory method here.. something like `Texture.FromFile` rather than `new Texture(string)` ... semantically speaking... creating in memory textures are fine with the ctor thing

Reply:
Not sure what you mean here?
I would like to go away from `Texture` and `Shader` objects, and instead use function to modify texture and shader objects.
But maybe that is not what you are saying, I'm not sure...

Reply:
I see what you mean now. Its not really clear that people want to get rid of `Texture` `Shader` objects etc... but if you prefer that, then line 80 should be `_texture = LoadTextureFromFile(...);` 

If however `Texture` as an object lives on in these tutorials, then I'd like to see a factory method of `Texture` to load one from a file, rather than using the `ctor` directly... ie `_texture = Texture.LoadFromFile(...);` .... Creating a texture from its ctor is not as semantically "obvious" as calling a method explicitly stating the intent (in this case loading it from a file).

I hope its a bit more clear now ;)

Reply:
I've made the change to `Texture.LoadFromFile(...)`.